ecosystem/sf-indexer/aptos-sf-indexer/proto/substreams.proto

```diff
@@ -0,0 +1,267 @@
+syntax = "proto3";
+
+package sf.substreams.v1;
+
+option go_package = "github.com/streamingfast/substreams/pb/sf/substreams/v1;pbsubstreams";
+
+import "google/protobuf/any.proto";
+import "google/protobuf/descriptor.proto";
+import "google/protobuf/timestamp.proto";
```

Comment:
we're never going to compile any of this to wasm, right?

Reply:
No.